persistence-utils/src/com/axonivy/market/jpa/demo/validation/IvyNotNull.java

```diff
@@ -0,0 +1,63 @@
+package com.axonivy.market.jpa.demo.validation;
```

Comment:
I would use the new namespace, e.g. ```com.axonivy.utils.persistence.validation```. This applies to the entire source code also in the demo and test project.